### PR TITLE
add ios label config

### PR DESCRIPTION
**Pull request by antonelli**
Don't merge until comments and delete are supported. :wink:

### CODE REVIEW

File: extras/label_configs/carrots-ios [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# unwanted labels
+delete:help wanted
+delete:invalid
+delete:wontfix
```
Comment (BrandonRomano) [inline extras/label_configs/carrots-ios]: @antonelli make sure you update this to be `-delete` after #21 is merged in!

**Comment by antonelli** [comment]
Done!

**Comment by BrandonRomano** [comment]
This gets my :+1:

**Comment by BrandonRomano** [comment]
Will have to refactor my Android one now that we have all these nice new features :cat:

**Pull request merged**